subsys/bluetooth/mesh/scene_srv.c

```diff
@@ -187,6 +260,10 @@ static void scene_recall(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ct
 	}
 
 	err = bt_mesh_scene_srv_set(srv, scene, has_trans ? &transition : NULL);
+	if (!err) {
```

Comment:
How will it work if the scheduler model recalls scenes? It should be integrated there or provided via the scene model API.